test/net/sourceforge/kolmafia/KoLAdventureValidationTest.java

```diff
@@ -918,6 +919,30 @@ public void canVisitBombedHippyCampAfterWossname() {
         assertTrue(zones.get(AdventurePool.BOMBED_HIPPY_CAMP).canAdventure());
       }
     }
+
+    @Test
+    public void changesIntoWarFratOutfitIfNecessary() {
+      setupFakeClient();
+
+      var cleanups =
+          new Cleanups(
+              withEquippableItem("beer helmet"),
+              withEquippableItem("distressed denim pants"),
+              withEquippableItem("bejeweled pledge pin"),
+              withQuestProgress(Quest.ISLAND_WAR, "step1"));
+
+      try (cleanups) {
+        var success = zones.get(AdventurePool.WARTIME_HIPPY_CAMP_DISGUISED).prepareForAdventure();
```

Comment:
Let's see canAdventure() first - which will fail, since you are At War. ;)

Or, better yet, change to QuestDatabase.STARTED.